tests/amethyst_test/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "amethyst_test"
+version = "0.1.0"
+authors = ["Azriel Hoh <azriel91@gmail.com>"]
+
+[dependencies]
+amethyst = { path = "../.." }
+boxfnonce = "0.1"
+derivative = "1.0"
+derive-new = "0.5"
+hetseq = "0.2"
+lazy_static = "1.1"
+
+[features]
+default = ["test_display"]
+nightly = ["amethyst/nightly"]
+
+# Used to tag tests that need a display (e.g. X) to run.
+test_display = []
```

Comment:
Can we call this `no_graphics` / `graphics`? Is a bit clearer hear, plus the `test_` is not required.

Reply:
haha you know, I spent a good 10 minutes trying to think of a name, `graphics` is good